pkg/apis/devops/v1alpha3/devopsproject_types.go

```diff
@@ -33,6 +33,7 @@ const (
 	DevOpeProjectSyncStatusAnnoKey = DevOpsProjectPrefix + "syncstatus"
 	DevOpeProjectSyncTimeAnnoKey   = DevOpsProjectPrefix + "synctime"
 	DevOpeProjectSyncMsgAnnoKey    = DevOpsProjectPrefix + "syncmsg"
+	DevOpsCredentialDataHash       = "devopscredential.devops.kubesphere.io/datahash"
```

Comment:
DevOpsCredentialDataHash should not be defined in devopsproject_types.go, but in credential_types.go. 

Please use DevOpsCredentialPrefix  + datahash replace "devopscredential.devops.kubesphere.io/datahash".

Keeping style consistent is more important than function.

Reply:
Sure.